test/e2e/tests/account/snap-account-transfers.spec.ts

```diff
@@ -44,6 +46,7 @@ describe('Snap Account Transfers', function (this: Suite) {
         );
         const headerNavbar = new HeaderNavbar(driver);
         await headerNavbar.checkAccountLabel('SSK Account');
+        await homePage.checkExpectedTokenBalanceIsDisplayed('25', 'ETH');
```

Comment:
If it clicks too fast before the balance is fetched, it does not detect there are funds.

A delay works, but checking for the updated balance is safer.